ocean_lib/structures/file_objects.py

```diff
@@ -49,6 +49,17 @@ def to_dict(self) -> dict:
         return {"type": self.type, "hash": self.hash}
 
 
+class ArweaveFile(FilesType):
+    @enforce_types
+    def __init__(self, transactionId: str) -> None:
+        self.transactionId = transactionId
```

Comment:
Can we keep snake_case convention for `transactionId`?

Reply:
Done.